Util/src/main/java/io/deephaven/util/thread/ThreadInitializationFactory.java

```diff
@@ -23,10 +24,20 @@ public interface ThreadInitializationFactory {
                     return clazz.getDeclaredConstructor().newInstance();
                 } catch (ClassNotFoundException | NoSuchMethodException | InvocationTargetException
                         | InstantiationException | IllegalAccessException e) {
+
+                    // TODO (https://github.com/deephaven/deephaven-core/issues/4040):
```

Comment:
Is this the right ticket?

Reply:
I believe so; I acquired it from Colin. It is the ticket related to python debug initialization. This snippet can be reverted once Colin takes an alternative approach.

Reply:
Yes, this is the ticket (in several parts...) that will address this.